recipe/meta.yaml

```diff
@@ -24,13 +24,17 @@ requirements:
   run:
     - python
     - numpy
+    - msgpack-python
+    - backports.lzma  # [py27]
 
 test:
   requires:
     - nose
   imports:
     - numcodecs
     - numcodecs.blosc
+    - numcodecs.lz4
```

Comment:
Oh sorry, for some reason I thought this said `lzma`. Obviously need more coffee. ☕️